installer/installer.py

```diff
@@ -406,43 +406,13 @@ def install(self):
             self.log(f"Moving binary from {self.binary_path} to {DEFAULT_INSTALL_PATH}")
             self.exec("sudo mv {} {}".format(self.binary_path, DEFAULT_INSTALL_PATH))
 
-        if self.interviewer.is_setup_needed:
-            self.post_install()
 
         if self.interviewer.init_from_snapshot:
             self.log("Downloading snapshot and extracting archive. This can take a *really* long time...")
             self.download_snapshot()
             self.untar_from_snapshot()
         self.print_success()
 
-    def post_install(self):
```

Comment:
Let's keep this step, cause such action is still needed. It should be moved to `sed` command calling or something like this. I think, that there is another task for `installer.py`

Reply:
@anikitinDSR Okay,thanks! I kept it empty now.

Reply:
Yes, we can't just get rid of post install here. I would suggest to replace `configure` command with regex based replace

Reply:
@anikitinDSR What do you think about regexps?